test/test_sparse_csr.py

```diff
@@ -2605,11 +2605,13 @@ def _to_from_layout(layout_a, layout_b):
                             or layout_b in [torch.sparse_csc, torch.sparse_bsc])
             expect_error = expect_error or (layout_a, layout_b) == (torch.sparse_bsr, torch.sparse_bsr)
             expect_error = expect_error or (layout_a, layout_b) == (torch.sparse_bsr, torch.sparse_csr)
-            # CSC to CSR conversion is supported
-            if layout_a is torch.sparse_csc and layout_b is torch.sparse_csr:
-                expect_error = False
-            # CSC to CSC conversion is supported
-            if layout_a is torch.sparse_csc and layout_b is torch.sparse_csc:
+
```

Comment:
If support is added for batch dims, this test would need to be expanded to generate samples with 3+ dims.

Reply:
On it.